howtos/how-to-archive-a-conversation/index.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+
+    <title>TalkJS tutorial</title>
+  </head>
+
+  <body>
+    <div id="chat-container" style="width: 90%; margin: 30px; height: 500px">
+      <button type="button" id="archive-button">View archived chats</button>
+      <!-- container element in which TalkJS will display a chat UI -->
+      <div id="talkjs-container" style="margin: 30px; height: 500px">
+        <i>Loading chat...</i>
+      </div>
+    </div>
+  </body>
+</html>
+
+<!-- minified snippet to load TalkJS without delaying your page -->
+<script>
+  (function (t, a, l, k, j, s) {
+    s = a.createElement("script");
+    s.async = 1;
+    s.src = "https://cdn.talkjs.com/talk.js";
+    a.head.appendChild(s);
+    k = t.Promise;
+    t.Talk = {
+      v: 3,
+      ready: {
+        then: function (f) {
+          if (k)
+            return new k(function (r, e) {
+              l.push([f, r, e]);
+            });
+          l.push([f]);
+        },
+        catch: function () {
+          return k && new k();
+        },
+        c: l,
+      },
+    };
+  })(window, document, []);
+</script>
+
+<script>
+  Talk.ready.then(function () {
+    const me = new Talk.User({
+      id: "archiveConversationExampleSupportAgent",
+      name: "Alice",
+      email: "alice@example.com",
+      role: "default",
+      photoUrl: "https://talkjs.com/images/avatar-1.jpg",
+      welcomeMessage: "Hey there! How can I help?",
+    });
+    const talkSession = new Talk.Session({
+      appId: "<APP_ID>", // replace with your app ID
+      me: me,
+    });
+
+    const inbox = talkSession.createInbox();
+    inbox.mount(document.getElementById("talkjs-container"));
+    inbox.onCustomConversationAction("archive", async (event) => {
+      let conv = talkSession.getOrCreateConversation(
```

Comment:
Maybe adding a comment here to explain why we're re-creating/selecting the current conversation would be useful? I know it's already in the tutorial, but <I think it could still be useful if someone's just looking at the example separately.

Reply:
Do you have a suggestion for what to add? I have this in the tutorial at the moment, but don't feel like I understand this particularly well.

https://github.com/talkjs/tutorials/blob/d732b4a8a6f5a4def0ff9611d4e10f42f7081b7e/how-to-archive-a-conversation/how-to-archive-a-conversation.md?plain=1#L57

Reply:
// TalkJS updates a conversatio's details when it's passed to one of our widgets to be displayed, 
// So we recreate the current conversation, witth updated custom values, and then pass it to our inbox widget to be synced.